sigmastate/src/main/scala/sigmastate/trees.scala

```diff
@@ -198,24 +201,30 @@ object SigmaAnd extends SigmaTransformerCompanion {
   */
 case class SigmaOr(items: Seq[SigmaPropValue]) extends SigmaTransformer[SigmaPropValue, SigmaPropValue] {
   override def companion = SigmaOr
-  def tpe = SSigmaProp
-  val opType = SFunc(SCollection.SSigmaPropArray, SSigmaProp)
+  override def tpe = SSigmaProp
+  override def opType = SigmaOr.OpType
 }
 
 object SigmaOr extends SigmaTransformerCompanion {
+  val OpType = SFunc(SCollection.SSigmaPropArray, SSigmaProp)
   override def opCode: OpCode = OpCodes.SigmaOrCode
   override def argInfos: Seq[ArgInfo] = SigmaOrInfo.argInfos
   def apply(head: SigmaPropValue, tail: SigmaPropValue*): SigmaOr = SigmaOr(head +: tail)
 }
 
+/** Base trait for companions. */
```

Comment:
This comment is not saying really anything.

Reply:
Well it is better then nothing. Added more details.